modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -543,28 +543,26 @@ def national_guard # rubocop:disable Metrics/MethodLength
         reserves = @pdf_data&.dig(:data, :attributes, :serviceInformation, :reservesNationalGuardService)
```

Comment:
curious as to this line, and now line 564. These were carried over from before, but they seem........ reversed?

```
564 - @pdf_data[:data][:attributes][:serviceInformation][:reservesNationalGuardService] = reserves
```